beamer/chain.py

```diff
@@ -473,3 +504,16 @@ def _withdraw(claim: Claim, context: Context) -> None:
 
     claim.transaction_pending = True
     log.debug("Withdrew", claim=claim.id, txn_hash=txn_hash.hex())
+
+
+def _invalidate(request: Request, claim: Claim, context: Context) -> None:
+    func = context.fill_manager.functions.invalidateFillHash(
+        request.id, request.request_hash, claim.latest_claim_made.fill_id, request.source_chain_id
```

Comment:
building on the latest main, request id is not a parameter of `invalidateFillHash`anymore